src/widgets/helper/EditableModelView.hpp

```diff
@@ -31,6 +31,8 @@ class EditableModelView : public QWidget
     QHBoxLayout *buttons_{};
 
     void moveRow(int dir);
+
+public:
```

Comment:
Not anything to be changed, just a note that I find this `public`, `private`, `public` setup a bit weird.